keyboards/mectechpad/keyboard.json

```
@@ -0,0 +1,42 @@
+{
+    "manufacturer": "Printronics",
+    "keyboard_name": "Mectechpad",
+    "maintainer": "jacksaxi",
+    "bootloader": "rp2040",
+    "debounce": 15,
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true
+    },
+    "matrix_pins": {
+        "rows": ["GP26", "GP15", "GP14"],
```

add the extra switch pin here
```suggestion
        "rows": ["GP26", "GP15", "GP14", "GP13"],
```